backend/private-graph/graph/resolver.go

```diff
@@ -76,6 +76,8 @@ const ErrorGroupLookbackDays = 7
 const SessionActiveMetricName = "sessionActiveLength"
```

Comment:
When reviewing, please also expand `backend/private-graph/graph/schema.resolvers.go` as github closed that file